pkg/console/operator/sync_v400.go

```diff
@@ -457,12 +457,30 @@ func (c *consoleOperator) ValidateCustomLogo(operatorConfig *operatorsv1.Console
 		klog.V(4).Infoln("no custom logo configured")
 		return false, "", nil
 	}
-	logoConfigMap, err := c.configMapClient.ConfigMaps(api.OpenShiftConfigNamespace).Get(logoConfigMapName, metav1.GetOptions{})
+
+	// original user-defined logo in openshift-config
+	_, err = c.configMapClient.ConfigMaps(api.OpenShiftConfigNamespace).Get(logoConfigMapName, metav1.GetOptions{})
 	// If we 404, the logo file may not have been created yet.
+	if apierrors.IsNotFound(err) {
+		msg := fmt.Sprintf("source custom logo file %v in openshift-config not found", logoConfigMapName)
+		klog.V(4).Infof(msg)
+		return false, "FailedGetSource", customerrors.NewCustomLogoError(msg)
+	}
+	if err != nil {
+		return false, "SourceError", customerrors.NewCustomLogoError(fmt.Sprintf("custom logo: %v\n", err))
+	}
+
+	// sync'd logo configmap into openshift-console namespace
+	logoConfigMap, err := c.configMapClient.ConfigMaps(api.OpenShiftConsoleNamespace).Get(api.OpenShiftCustomLogoConfigMapName, metav1.GetOptions{})
```

Comment:
This is what we did not do previously.  We need the source configmap, but also the destination confimap validated.